src/misc/ClientDetector.ts

```diff
@@ -24,7 +24,7 @@ export class ClientDetector {
 
 		this._setDeviceInfo()
 
-		this.overflowAuto = this.cssPropertyValueSupported("overflow", "overlay") ? "overlay" : "auto"
+		this.overflowAuto = "auto"// this.cssPropertyValueSupported("overflow", "overlay") ? "overlay" : "auto"
```

Comment:
straggling comment

Reply:
oh we shouldn't actually change it at all, oops